bamboo/compiler_tests/conftest.py

```diff
@@ -2,9 +2,15 @@
 
```

Comment:
I think this configuration file needs the same lines from unit tests' which determine what cluster we are on.

Reply:
Fixed.

Reply:
need to import re